eos/b-decays/b-to-3l-nu.cc

```diff
@@ -130,34 +130,34 @@ namespace eos
             / (9. * k2 * m_B_sq);
             const double f12 = (128 * (2 * power_of<2>(k2) - k2 * m_l_sq - m_l_4) * M_PI * q2
             * (k2 - m_B_sq + q2) * (2 * m_lprime_sq + q2)) / (3. * k2 * m_B_sq);
-            const double f15 = (256 * m_l_sq * M_PI * (2 * m_lprime_sq + q2) * (- ((k2 - m_l_sq)
+            const double f15 = -(256 * m_l_sq * M_PI * (2 * m_lprime_sq + q2) * (- ((k2 - m_l_sq)
             * (2 * k2 - m_B_sq + m_l_sq + q2) * sqrt(power_of<2>(k2) + power_of<2>(m_B_sq - q2)
             - 2 * k2 * (m_B_sq + q2))) + 2 * k2 * (k2 - m_B_sq) * (k2 - m_B_sq + 2 * m_l_sq + q2)
-            * atan2(((k2 - m_l_sq) * sqrt(power_of<2>(k2) + power_of<2>(m_B_sq - q2) - 2 * k2
-            * (m_B_sq + q2))),(power_of<2>(k2) + m_l_sq * (- m_B_sq + q2) - k2
+            * std::atanh(((k2 - m_l_sq) * sqrt(power_of<2>(k2) + power_of<2>(m_B_sq - q2) - 2 * k2
```

Comment:
This commit needs a bit more documentation:
 - replaced incorrect function call to ``atan2`` with ``atanh``;
 - fixed sign errors in the terms proportional to ``f_B``